### DISCUSSION TITLE
Transform Words into Sentences and Paragraphs?

### DISCUSSION THREAD

**Discussion by GlennIM** [sergey-tihon/Stanford.NLP.NET#109] [Q&A]
Is it possible to use this library to create sentences and paragraphs from a group of words. Here is an example group of words.

"hi there my name is Arthur Wilder I am four weeks into taking this pill and I'm glad you're here looking at this video because I believe what's getting ready to happen is your life is gonna change because you'll do what I did is you trilemma nine it has made me happier it's called the happy pill it has made me calmer has made me more focused I have more energy I sleep better and I'm so grateful for this product I have a lot of things that the company says it does that I'm seeing it doing my body it's kind of touching a whole bunch of areas it doesn't affect your libido it does affect your stamina it does affect your workout recovery and that's what the clinical studies say and I'm finding that as the truth and I'm not the only one that's feeling that have almost a hundred people in my downline after four weeks the companies paid me over a thousand dollars which is great "

**Comment by sergey-tihon** [comment]
Not sure, better to ask on SO - https://stackoverflow.com/questions/tagged/stanford-nlp
I think that this is not classic NLP problem. You may take a look here
- https://linguistics.stackexchange.com/questions/3167/are-there-sentence-boundary-disambiguation-algorithms-which-can-handle-punctuati
- https://www.quora.com/Natural-Language-Processing-What-are-the-ways-to-do-Sentence-Boundary-Detection-in-English
